lib/tasks/test_self_service.rake

```diff
@@ -1,6 +1,5 @@
 namespace :test do
   namespace :self_service do
-    desc "Setup environment for self_service tests"
```

Comment:
I explained this poorly off my phone; see https://github.com/chrisarcand/manageiq/commit/c6a315ff82198f72f2daef3780c1a08fd24bad5f message for a better explanation.